io/net/bonding.py.data/bonding.yaml

```diff
@@ -16,6 +16,7 @@ Test: !mux
 bond_interfaces: ""
 host_ips: ""
 netmask: ""
+peer_ips:
 peer_ip: ""
```

Comment:
why keep both,  have peer_ips only,,, and handle in code if the user gives singel or 2 peer ip name

Reply:
@abdhaleegit  Keeping on one parameter for peer_ip . It can have one or two interface ips. Which will work either way.